src/AddTodoForm.jsx

```diff
@@ -1,8 +1,19 @@
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        
+        const todoTitle = document.querySelector('[name="title"]').value;
```

Comment:
The form inputs are attached directly to the event, so a safer way to call this is `event.target.title.value`